stdlib/public/SDK/Foundation/NSArray.swift

```diff
@@ -31,7 +31,7 @@ extension Array : _ObjectiveCBridgeable {
   /// The provided `NSArray` will be copied to ensure that the copy can
   /// not be mutated by other code.
   internal init(_cocoaArray: NSArray) {
-    _sanityCheck(_isBridgedVerbatimToObjectiveC(Element.self),
+    precondition(_isBridgedVerbatimToObjectiveC(Element.self),
```

Comment:
This seems like an expensive upgrade (for most of the Foundation things). `assert` is probably fine, given that `_sanityCheck` would have been compiled out in a release build too.

Reply:
yeah, that's the better call.